### PR TITLE
* ArrowTaggedActivity Improvements / fixes

### PR DESCRIPTION
**Pull request by fnk0**
* Changed the ArrowTaggedActivity from ListView to RecyclerView
* Made the list items follow more of the design guidelines and changed cards to simple list items
* Fixed a bug that would keep making the list get populated if user hit home button and them came back to the app
* Reduced 1 layer of Overdraw by moving from CardView to a normal list
* Improved overall look of the list

#### Before (Cards with Overdraw)
before

#### After (List items without overdraw)
after

after_nogpu

### CODE REVIEW

**Comment by tasomaniac** [comment]
The only thing I notice is that you are not using dimen set for image size. Other than that it looks perfect.

**Comment by fnk0** [comment]
@tasomaniac sorry! In all those git stuff things got messed up. I'll fix that asap and push a commit

**Comment by tasomaniac** [comment]
Thank you for the contribution.

File: app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java [hunk 1]
```diff
@@ -61,10 +62,12 @@
     public static final String ID_SPLIT_CHAR = "|";
 
     @Bind(R.id.taggedList)
-    ListView taggedList;
+    RecyclerView taggedList;
     String taggedOrganizers = "";
     private OrganizerAdapter adapter;
 
+    String[] orgas;
```
Comment (Splaktar) [inline app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java]: I much prefer something like `organizers` to an abbreviated name like `orgas`.

Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java]: `taggedOrganizers` is way to go.

**Comment by fnk0** [comment]
There's another variable named ```taggedOrganizers``` as well.

File: app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java [hunk 2]
```diff
@@ -120,7 +118,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
 
                     @Override
                     public void success(final Directory directory, final retrofit.client.Response response) {
-                        String[] orgas = taggedOrganizers.split("\\|");
+                        if (orgas != null) {
+                            return;
+                        }
+                        orgas = taggedOrganizers.split("\\|");
```
Comment (Splaktar) [inline app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java]: This should be using `ID_SEPARATOR_FOR_SPLIT`

Comment (fnk0) [inline app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java]: Oh. Probably. I just left that how it was before. (Still getting to know the project better. I have read quite a bit of the overall code but not everything yet!)

Comment (Splaktar) [inline app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java]: Yep, no problem. This was a bit of a mess when you started. I'm trying to submit a PR tonight that should help clean it up some.

**Comment by Splaktar** [comment]
Take a look at #455 and let me know if that makes more sense or if I screwed anything up :)

**Comment by tasomaniac** [comment]
You cannot screw something up at that screen. :)

**Pull request merged**